plugins/plugin_repeat.py

```diff
@@ -13,17 +13,8 @@ def __init__(self,slack_client=None, plugin_config=None):
             slack_client=slack_client,
             plugin_config=plugin_config
         )
-        self.words = [
-            'banana',
-            'bananas',
-            'banana\'s',
-            'apple',
-            'apples',
-            'apple\'s',
-            'pineapple',
-            'pineapples',
-            'pineapple\'s'
-        ]
+        with open("words.txt") as f:
```

Comment:
I would generally define this in a sep function that `__init__()` could call, so that we can reuse it elsewhere if we have more that one list to load.